content/docs/dvclive/how-to/checkpoints.md

```diff
@@ -0,0 +1,42 @@
+# Integrate with DVC Checkpoints
+
+When using [DVC Checkpoints](/doc/user-guide/experiment-management/checkpoints)
+and/or enabling DVCLive's [`resume`](/doc/dvclive/api-reference/live#parameters)
+you need to add the flag
+[`persist: true`](/doc/user-guide/project-structure/pipelines-files#output-subfields)
+to all DVCLive outputs.
+
+Adding `--type checkpoint` to `dvc exp init` will take care of doing this when
+generating the `dvc.yaml`:
+
+```dvc
+$ dvc exp init \
```

Comment:
Is this the recommended "happy path" method? The intro above is already a bit complicated conceptually (mentioning checkpoints and output persistence), so introducing also experiments isn't ideal. If possible move it after the YAML sample as a tip/ alternative easy way to get there.

Reply:
I changed the order. As commented above the scope here was a concise explanation on how to set up for resume training, not to introduce the concepts to dvclive users

Reply:
Thanks for addressing my feedback. Sorry I missed the updates until now...